src/Sylius/Bundle/CurrencyBundle/Resources/translations/validators.en.yml

```diff
@@ -9,6 +9,7 @@ sylius:
     exchange_rate:
         ratio:
             greater_than: The ratio must be greater than 0.
+            less_than: The ratio must be less than 100000.
```

Comment:
We should use actual values passed from the validator rather than hardcoding them.
Same case w/ `greater_than`.